Formula/mpc.rb

```diff
@@ -20,9 +20,13 @@ def install
     system "meson", "--prefix=#{prefix}", ".", "output"
     system "ninja", "-C", "output"
     system "ninja", "-C", "output", "install"
+
+    bash_completion.install "contrib/mpc-completion.bash" => "mpc"
+    rm share/"doc/mpc/contrib/mpc-completion.bash"
```

Comment:
Why remove it from the doc folder if it wants it there? Does it hurt to keep it?